scripts/direct-db-setup.js

```diff
@@ -62,6 +62,9 @@ async function setupDatabase() {
       if (files.includes('_001_init.sql')) {
         migrationFiles.push('_001_init.sql');
       }
+      if (files.includes('002_api_keys.sql')) {
+        migrationFiles.push('002_api_keys.sql');
+      }
```

Comment:
this might be some vibe code weirdness. Let's fix it so we don't have to ever edit the db-setup file when we create a new migration file.